### ISSUE TITLE
Test database?

### ISSUE THREAD

**Issue by sinnbeck** [sinnbeck/laravel-served#17]
A test database would be nice. Should it be created by default or be an option (the user can add one manually, but that is more work)

**Comment by iammikek** [comment]
We're not gaining much overhead by adding an empty database, so adding it by default and it not being used wouldn't be too bad.

**Comment by sinnbeck** [comment]
Ok so we should just add a small sql script to create an extra database in Images/stubs.

Do you have experience with postgres?

**Comment by iammikek** [comment]
I've worked with Postgres before, but not for a while. what are you looking to do?

**Comment by sinnbeck** [comment]
Just wondering if we can create a database with the same sql command as for mysql. And if there are anything important to consider.

Otherwise I will just Google it 👍

**Comment by iammikek** [comment]
I'd have to google it to remind myself TBH.

**Comment by sinnbeck** [comment]
I'll brainstorm a bit. Maybe instead allow the user to add a db with an artisan command?

**Comment by iammikek** [comment]
running an extra artisan command would be my preference. Was thinking that we could have a command like the following 
```
php artisan served:config
```
to do this, and then we can prompt for some details.

**Comment by sinnbeck** [comment]
Yeah. Or something database specific like ` php artisan served:db-manage`

**Comment by sinnbeck** [comment]
But maybe for a start it is OK with external access, and "shell" access

```
php artisan served:ssh mysql
